### ISSUE TITLE
deleted features with geometrycolletion type returned as if they were not deleted

### ISSUE THREAD

**Issue by tyrasd** [GIScience/ohsome-api#131]
for example: [relation/2245739](https://www.openstreetmap.org/api/0.6/relation/2245739/history) was deleted on `2018-10-20T18:32:04Z`. However, it is returned "like" a not-deleted element in the `contributions` endpoint:

```bash
$ curl -X GET "https://api.ohsome.org/v1/contributions/geometry?bboxes=11.2162%2C46.4768%2C11.2931%2C46.5353&clipGeometry=true&filter=type%3Arelation%20and%20type%3Drestriction%20and%20id%3A2245739&properties=tags&time=2012-01-01%2C2020-01-01" -H  "accept: application/json"
… [snip] …
  }, {
    "type" : "Feature",
    "geometry" : {
      "type" : "GeometryCollection",
      "geometries" : [
… [snip] …
      ]
    },
    "properties" : {
      "@osmId" : "relation/2245739",
      "@timestamp" : "2018-10-20T18:32:04Z"
    }
  }]
}
```

I would have expected the following instead:

```json
  }, {
    "type" : "Feature",
    "geometry" : null,
    "properties" : {
      "@osmId" : "relation/2245739",
      "@timestamp" : "2018-10-20T18:32:04Z"
    }
  }]
}
```

possibly related issue: #129 ?

**Comment by FabiKo117** [comment]
I don't quite get the point here. Is this not the same as described in the other issue?

**Comment by tyrasd** [comment]
well, it's worse, since for `Point`, `LineString` and `Polygon` geometries one gets at least an empty coordinate array to be able to distinguish deleted versions from regular edits. But here, a deleted element is returned as if nothing happened.

**Comment by tyrasd** [comment]
PS: But I agree in that the fix for #129 (#130) will likely also fix this problem as well.

**Comment by FabiKo117** [comment]
It seems like this issue persists with the added fixes in #130.

Will further investigate using the following example:
`http://localhost:8080/contributions/geometry?bboxes=8.625%2C49.3711%2C8.7334%2C49.4397&clipGeometry=true&filter=type%3Arelation%20and%20type%3Drestriction&properties=tags%2Cmetadata&time=2018-01-01,2019-01-01`
